### ISSUE TITLE
Trouble Launching Game From Source for Development

### ISSUE THREAD

**Issue by rahulsurti97** [OpenEmu/OpenEmu#4673]
# TL;DR
I keep running into the popup error, "Timeout waiting for connection from helper app" when trying to open any game.  I've followed the directions for compiling from source with cores.

# Past issues
I've found an identical issue in the past, however, it seemed like the previous user was trying to compile from source to play games.  I am trying to compile from source to develop and add a feature.
https://github.com/OpenEmu/OpenEmu/issues/4376

I've ran both the `xattr` and `launchctl` commands with the path to the app in the Release folder and to the applications folder after moving it, with no success

# Trace
I've traced the issue to`loadROMWithCompletionHandler`, where it was unable to create a helperConnection to `org.openemu.broker`

**Comment by rahulsurti97** [comment]
Could compiling from source be an issue with the M1 Mac on Big Sur?

**Comment by stuartcarnie** [comment]
Compiling under M1 will cause this issue as you are likely mixing two architectures. The cores (when downloaded from the cloud) will be x86 architecture and OpenEmu under Xcode will default to running using arm64. Two options for you:

* Run OpenEmu using Rosetta in Xcode
* Compile a core using arm64 and use that core to test

**Comment by brunocastello** [comment]
I have got all my desired cores except PPSSPP to compile for M1 arm64, I cant seem to find how to solve that one. I'm pretty close to a full native build...
